server/RenderContext.cs

```diff
@@ -626,9 +626,37 @@ public void Render(XGraphics graphics)
                     }
                 }
                 timers.Add(new Timer("droyne"));
-#endregion
+                #endregion
+
+                #region droyne
+                //------------------------------------------------------------
+                // Non-Hiver Worlds
+                //------------------------------------------------------------
+                if (styles.nonHiverWorlds.visible)
+                {
+                    solidBrush.Color = styles.nonHiverWorlds.textColor;
+                    foreach (World world in selector.Worlds)
+                    {
+                        if (world.HasCode("Nh"))
```

Comment:
We may want to make this smarter, to avoid data changes; I don't think `Nh` is what we'd use in T5SS data. Rather, we could check for `HvFd` allegiance and add some generic sophont check to `World`? Even `world.HasCodePrefix("(")` might be sufficient.